frontend/src/styles/LoginRegistrationPage.css

```diff
@@ -1,97 +1,246 @@
-div.loginHeader {
-  font-size: 40px;
-  color: #1DE28F;
-}
+@import url('https://fonts.googleapis.com/css?family=Teko');
 
+:root {
+  --BackgroundColor: #202020;
+  --themeColor: #1DE28F;
+  --bodyColor: var(--baseColor);
+  --FontFamily: 'Teko';
+  --FontSize: 14px;
+  --FontWeight: 300;
+  --LineHeight: 1.5;
+}
+body {
```

Comment:
do not define css for body, html, or anything general as it can override other styles